mineral-ui/mineral-ui
# [Website]: add <title> to document

connects #427 

### Description
- use React Helmet to add a unique title and canonical link to each page.
- add `lang` attribute

### Motivation and context
Since we have a single-page app, our page title never changed on navigation. This hurts SEO and accessibility. Now the title changes to give hints upon navigation.

https://document-title--mineral-ui.netlify.com/

### How to test

1. `cd mineral-ui && npm start`
2. Navigate to all the pages. the format should be `title | Mineral UI`
3. `<html>` element should have a `lang` attribute set.

### Types of changes
- New feature (non-breaking change which adds functionality)

### Checklist




### How does this PR make you feel?

![open for business](https://media.giphy.com/media/nKxr0WXJATmoM/giphy.gif)
